deploy/helms/event-collector/values.yaml

```diff
@@ -0,0 +1,31 @@
+rootDomain: ""
+localDomain: "cluster.local"
+domainDelimiter: "-"
+
+replicas: 1
+ingress:
+    annotations: {}
+    domain:
+      partial: feedback
+    tls:
+      enabled: true
+      autoSecretNameDeduction: true
+
+image: "legion/k8s-fluentd:latest"
+PullPolicy: "Always"
+
+s3:
+  bucket: "none"
+  region: "us-east-2"
```

Comment:
@alexander-semenets, s3 bucket name should in profile configuration rather than hardcoded in helm values

Reply:
This file stores default configuration for HELM chart

Reply:
yes, this is just default values to show which parameters are required for the chart.
Dropped both to "none"